src/tools/PrizeTierController/config.ts

```diff
@@ -22,3 +22,66 @@ export const PRIZE_TIER_CONTROLLER_SUPPORTED_CHAIN_IDS = Object.freeze({
     CHAIN_ID.fuji
   ]
 })
+
+export const PRIZE_TIER_HISTORY_V1: { [chainId: number]: { address: string; token: string } } =
+  Object.freeze({
+    [CHAIN_ID.optimism]: { address: '0xC88f04D5D00367Ecd016228302a1eACFaB164DBA', token: 'USDC' },
+    [CHAIN_ID.polygon]: { address: '0x1DcaD946D10343cc4494D610d6273153FB071772', token: 'USDC' },
+    [CHAIN_ID.mainnet]: { address: '0x24C3e15BdC10Ce2CB1BEc56cd43F397cE9B89430', token: 'USDC' },
+    [CHAIN_ID.avalanche]: { address: '0xC3DAD539E460103c860Bb9Ca547647EDbD4903b6', token: 'USDC' },
+    [CHAIN_ID['optimism-goerli']]: {
+      address: '0xF567588A82660F9F93059E97063360900387a2cc',
+      token: 'USDC'
+    },
+    [CHAIN_ID['arbitrum-goerli']]: {
+      address: '0x6A501383A61ebFBc143Fc4BD41A2356bA71A6964',
+      token: 'USDC'
+    },
+    [CHAIN_ID.mumbai]: { address: '0x0450Be4f1e986Ef22D01d00d75dcb593E6840057', token: 'TOK' },
+    [CHAIN_ID.goerli]: { address: '0xF567588A82660F9F93059E97063360900387a2cc', token: 'USDC' },
+    [CHAIN_ID.fuji]: { address: '0x145d38344fb8d35606b221F513d2BaEa2691c029', token: 'TOK' }
+  })
+
+// export const PRIZE_TIER_HISTORY_V2: { [chainId: number]: { address: string; token: string } } =
+//   Object.freeze({
+//     [CHAIN_ID.optimism]: { address: '', token: 'USDC' },
+//     [CHAIN_ID.polygon]: { address: '', token: 'USDC' },
+//     [CHAIN_ID.mainnet]: { address: '', token: 'USDC' },
+//     [CHAIN_ID.avalanche]: { address: '', token: 'USDC' },
+//     [CHAIN_ID['optimism-goerli']]: {
+//       address: '',
+//       token: 'USDC'
+//     },
+//     [CHAIN_ID['arbitrum-goerli']]: {
+//       address: '',
+//       token: 'USDC'
+//     },
+//     [CHAIN_ID.mumbai]: { address: '', token: 'TOK' },
+//     [CHAIN_ID.goerli]: { address: '', token: 'USDC' },
+//     [CHAIN_ID.fuji]: { address: '', token: 'TOK' }
+//   })
+
+export const TOKEN_INFO: {
```

Comment:
Lets stick to using chain id's and token addresses as keys so we don't run into any collisions/confusion in the future
`PRIZE_TIER_HISTORY_TOKENS: { [chainId: number]: { [address: string]: Token } }` where `Token` is the common type from `@pooltogether/hooks`

Reply:
Updated as recommended to use the existing `Token` interface: a2bde40